Monika After Story/game/zz_submods.rpy

```diff
@@ -6,17 +6,39 @@ init 10 python:
     store.mas_submod_utils.Submod._checkUpdates()
 
 init -989 python:
+    #Log initialized submods
+    if store.mas_submod_utils.submod_map:
+        store.mas_submod_utils.writeLog(
+            "INSTALLED SUBMODS: {0}\n".format(
+                ", ".join(
+                    ["'{0}' v{1}".format(submod.name, submod.version) for submod in store.mas_submod_utils.submod_map.itervalues()]
```

Comment:
Just some clarity here for those who have a *lot* of submods installed.
```suggestion
                    ["    '{0}' v{1}\n".format(submod.name, submod.version) for submod in store.mas_submod_utils.submod_map.itervalues()]
```

Reply:
Are you sure? That'll increase the number of lines we write each sesh, and since we're never going to clear the file. Eventually it'll be quite slow to read/write.